app/constants/ActionTypes.js

```diff
@@ -29,28 +29,29 @@ export const SET_VERSION_NUMBER = 28;
 export const SET_PRODUCT_CODE = 29;
 export const CHANGE_UPDATE_PROGRESS = 30;
 export const SET_LOGIN_TOKEN = 33;
-export const SET_CONSOLE_ACTIVE = 34;
-export const SET_SERVER_DOWNLOAD_PROGRESS = 35;
-export const SORT_SERVERS_BY = 36;
-export const CYCLE_SERVER_SORT_DIRECTION = 37;
-export const EXECUTE_CONSOLE_COMMAND = 38;
-export const CLEAR_CONSOLE = 39;
-export const SET_AVAILABLE_XPACKS = 40;
-export const SET_VEXT_VERSION = 41;
-export const SET_MIN_SERVER_BUILD = 42;
-export const SET_SETTINGS = 43;
-export const SET_CURRENT_SETTINGS = 44;
-export const SET_SERVER_FILTERS = 45;
-export const SET_PENDING_SERVER = 46;
-export const SET_VOIP_DATA = 47;
-export const SHOW_SETTINGS_POPUP = 48;
-export const SET_NEWS = 49;
-export const SET_GLOBAL_NOTICE = 50;
-export const SET_ACCOUNT_STORAGE = 51;
-export const SET_FAVORITE_SERVERS = 52;
-export const SET_MOD_SETTINGS = 53;
-export const SET_ACCOUNT_STORAGE_VALUE = 54;
-export const REMOVE_ACCOUNT_STORAGE_VALUE = 55;
-export const ADD_FAVORITE_SERVER = 56;
-export const REMOVE_FAVORITE_SERVER = 57;
-export const TOGGLE_FAVORITE_SERVERS_ONLY = 58;
+export const SET_LOGIN_DATA = 34;

```

Comment:
Always add to the end. Never shift numbers.

Reply:
done